uportal-war/src/main/resources/properties/ehcache.xml

```diff
@@ -1910,10 +1910,28 @@
            eternal="false" overflowToDisk="false" diskPersistent="false"
            maxElementsInMemory="100" timeToIdleSeconds="60" timeToLiveSeconds="60" memoryStoreEvictionPolicy="LRU" statistics="true"/>
 
-    <!-- PAGS Group Definition Cache, not replicated -->
-    <cache name="org.jasig.portal.groups.pags.dao.EntityPersonAttributesGroupStore"
+    <!--
+     | Stores objects of type IEntityGroup (concrete class EntityGroupImpl), which are
+     | the things emitted by this flavor of GaP (or any flavor) and used by the rest
+     | of the portal.  Historically, it was a big problem if multiple copies of the same
+     | logical group existed simultaneously in memory.  Due to recent changes, that
+     | shouldn't be a major issue (but it's still a good idea to cache them).  Not
+     | replicated.
+     +-->
```

Comment:
The style of documentation that gives in indication of the cardinality would be useful and consistent here e.g. - 1 x user logins